src/view/screens/Home.tsx

```diff
@@ -51,6 +52,8 @@ function HomeScreenReady({
   preferences: UsePreferencesQueryResponse
   pinnedFeedInfos: FeedSourceInfo[]
 }) {
+  useOTAUpdates()
```

Comment:
Moving this to `Home.tsx` because Statsig has not loaded if we call this hook immediately in `App.native.tsx`. Running it in `HomeScreenReady` ensures that it has loaded.